content/docs/SUSHI/running/_index.md

```diff
@@ -66,20 +70,22 @@ Here are some general tips for debugging:
 
 ## SUSHI Outputs
 
-Based on the inputs in FSH files, **config.yaml**, and the **ig-data** directory, SUSHI populates the output directory. For example, the customized-ig project from the [Project Structure](/docs/sushi/project/) section would result in output like the following:
+Based on the inputs in FSH files, **sushi-config.yaml**, and the IG project directory, SUSHI populates the output directory. For example, the customized-ig project from the [Project Structure](/docs/sushi/project/) section would result in output like the following:
 
 ```text
 customized-ig
-├── fsh
-│   └── (fsh files)
+├── fsh-generated
+|   ├── CodeSystem-myCodeSystem.json
```

Comment:
All these files in `fsh-generated` should be nested in a `resources` subdirectory also.